### DISCUSSION TITLE
Allow options with same values

### DISCUSSION THREAD

**Discussion by osaton** [orchidjs/tom-select#210] [Ideas]
I have a case where I need to list options with same value.

```
<select>
    <option value="chili">Carolina reaper</option>
    <option value="chili">Habanero</option>
</select>
```

Now only one `chili` option is shown. It would make sense to show all as that is how native select works?

Basically same as this issue that never got resolved: https://github.com/selectize/selectize.js/issues/597

**Comment by Mepcuk** [comment]
@osaton why you need the same option value for many option ? Can you describe your case?

**Comment by iamB0rgy** [comment]
> I have a case where I need to list options with same value.
> 
> ```
> <select>
>     <option value="chili">Carolina reaper</option>
>     <option value="chili">Habanero</option>
> </select>
> ```
> 
> Now only one `chili` option is shown. It would make sense to show all as that is how native select works?
> 
> Basically same as this issue that never got resolved: [selectize/selectize.js#597](https://github.com/selectize/selectize.js/issues/597)

Just add this parameter, `duplicates: true,`

**Reply by ajgarlag** [reply]
I can not get it to work. Can you help me?

See my [jsfiddle](https://jsfiddle.net/g9wn60e1/), please.
